cmd/clairctl/delete.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"os"
+
+	"github.com/quay/clair/v4/internal/httputil"
+	"github.com/quay/claircore"
+	"github.com/urfave/cli/v2"
```

Comment:
```suggestion
	"github.com/quay/claircore"
	"github.com/urfave/cli/v2"

	"github.com/quay/clair/v4/internal/httputil"
```

Reply:
Huh, I guess goimports gets confused when the cmd is different